features/battery.yml

```diff
@@ -0,0 +1,16 @@
+name: Battery status
+description: The `navigator.getBattery()` API obtains information about the device's battery, including charge level and status. Also known as the Battery API.
+spec: https://w3c.github.io/battery/
+status:
+  compute_from: api.BatteryManager
```

Comment:
This is not wrong exactly but, in general, prefer `compute_from` values that are the most-essential entry point(s) to the feature, what a developer would actually have to type to use it. Here that's the `navigator` API, not the `BatteryManager` interface.

(The exception is when you need to align with caniuse—where you'd use the minimal set of keys to achieve that, assuming the entry point is not sufficient—but that's not needed here.)

```suggestion
  compute_from: api.Navigator.getBattery
```